dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -1,10 +1,10 @@
-.multi{id: "multi_#{index}"}
+- data = level.properties
+- app = 'multi'
 
-  - data = level.properties
-  - app = 'multi'
+- # setting this so that multi_t works
```

Comment:
it seems a bit strange to have to do this. Now that we have multiple multis (:p) on a page maybe multi_t should start taking the level as an argument instead of using @level